modules/manage/pages/manage-nodes/manage-address-families.adoc

```diff
@@ -37,6 +37,8 @@ Raw IP addresses _can_ be used to name cluster-nodes, but the cluster must be _c
 
 * Node-to-node encryption must be disabled prior to the changeover (and can be re-enabled _after_ the changeover).
 
+* Subsequent to the changeover, a _restart_ is required for _all services except the Data Service_.
```

Comment:
As we discussed in slack: Restart of services (except the Data Service) will happen during changeover